libs/fluid-elements/lit-elements-loader.js

```diff
@@ -0,0 +1,40 @@
+/**
```

Comment:
do we want to move this file more to the workspace library as it should be consumed by others I think as well? (every app/ library that consumes the elements or?)